packages/discovery/src/dns/constants.ts

```diff
@@ -1,9 +1,12 @@
 import type { NodeCapabilityCount } from "@waku/interfaces";
 
 export const enrTree = {
-  TEST: "enrtree://AOGYWMBYOUIMOENHXCHILPKY3ZRFEULMFI4DOM442QSZ73TT2A7VI@test.waku.nodes.status.im",
-  SANDBOX:
-    "enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im"
+  DEFAULT_PUBSUB:
```

Comment:
I'd name it `DEPRECATED` as to clearly show what is going to happen to this. Maybe `DEPRECATED_DEFAULT_PUBSUB`.

As for others I think we can keep `SANDBOX` and `TEST` as it `TWN` is default behavior and can change later on (rebranding e.g).